packages/core-typings/src/voip/IVoipClientEvents.ts

```diff
@@ -0,0 +1,54 @@
+export type VoipPropagatedEvents =
+	| 'agentcalled'
```

Comment:
is this a good moment to rename these events to make them easier to read? 😬 
like `agent-called` or `queue-member-added`

Reply:
😬 well... it won't affect anyone to change it lol